pkg/storaged/details.js

```diff
@@ -1419,13 +1419,21 @@ define([
                 };
             }
 
+            var degraded_message = null;
+            if (mdraid.Degraded > 0) {
+                if (mdraid.Degraded > 1)
```

Comment:
This should use `cockpit.ngettext`.  See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html for a lecture.